test/e2e-api/content/posts.test.js

```diff
@@ -309,4 +309,18 @@ describe('Posts Content API', function () {
 
         tiersPostData.tiers.length.should.eql(1);
     });
+
+    it('Can use post excerpt as field', async function () {
+        await agent
+            .get(`posts/?fields=excerpt&limit=1`)
```

Comment:
not sure about `&limit1` here. we should be just fine requesting all the posts we have. the upside of not limiting the response - we can monitor excerpt field regressions if they happen